src/sass/style.scss

```diff
@@ -144,6 +144,21 @@ section#call-for-papers {
   }
 }
 
+section#map-information {
+  .content {
+    text-align: center;
+  }
+}
+
+section#map {
+  min-width: 200px;
+  width: 100%;
+  #map-canvas {
+    width: 100%;
+    height: 500px;
```

Comment:
500px is way to big for mobile device.

![screen shot 2015-05-29 at 20 21 03](https://cloud.githubusercontent.com/assets/2368417/7889516/43129ae2-0640-11e5-864b-18c1def54995.png)

Also make map undraggable at least for mobile devices.